tests/validator/validator.cpp

```diff
@@ -2018,14 +2026,13 @@ TEST(Validator, unitStandardUnitsWhichAreBaseUnits)
 
     validator->validateModel(m);
 
-    EXPECT_EQ_ERRORS(expectedErrors, validator);
+    EXPECT_EQ_ISSUES(expectedIssues, validator);
 }
 
 TEST(Validator, unitStandardUnitsWhichAreNotBaseUnits)
 {
-    const std::vector<std::string> expectedErrors = {
-        "Variable 'v1' has units of 'litre' and an equivalent variable 'v2' with non-matching units of 'gram'. The mismatch is: kilogram^-1, metre^3.",
-    };
+    const std::vector<std::string> expectedIssues = {
+        "Variable 'v1' in component 'c1' has units of 'litre' and an equivalent variable 'v2' in component 'c2' with non-matching units of 'gram'. The mismatch is: kilogram^-1, metre^3."};
```

Comment:
Add a comma at the end of the issue string, so that ClangFormat can format this better.

Reply:
Done